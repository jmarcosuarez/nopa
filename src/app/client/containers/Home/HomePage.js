import React, { PropTypes } from 'react';
import { Link } from 'react-router';
import { Layout, Button } from '../../components';
import * as Paths from '../../constants/paths';
import styles from './Home.css';

const HomePage = props =>
  <Layout title="Welcome to Nopa!">
    <div className={styles.main}>
      <div className={styles.logo}>
        <Link to={Paths.HOME} title="Home" className="">
          <img className="Nopa" alt="Logo" src={require('../../../static/images/Logo_Nopa.svg')} />
        </Link>
      </div>

      <h1>Your finances, in one place</h1>
      <p>Track all of your payments by connecting as many bank accounts as<br />
           you&#39;d like to your Nopa account and get updates on your balance instantly.</p>
           
      <div className={styles.button}>
        <Button to={Paths.CHOOSE_BANK} onClick={props.onGetStatedButtonClick}>Get started</Button>
      </div>
    </div>

    <div className={styles.secondary}>
      <div>
        <div>
          <h1>
            There is no such things as &quot;one size fits all&quot; finance.
          </h1>
          <p>
            We were founded to make money simple and fair for everyone:
            whether you are looking for a loan, or to get better rewards for your investments.
          </p>
        </div>
        <div>
          <img className={styles.shapes} alt="Shapes" src={require('../../../static/images/Shapes.png')} />
        </div>
      </div>
    </div>

  </Layout>;

export default HomePage;

HomePage.propTypes = {
  onGetStatedButtonClick: PropTypes.func,
};

HomePage.defaultProps = {
  onGetStatedButtonClick: () => {},
};