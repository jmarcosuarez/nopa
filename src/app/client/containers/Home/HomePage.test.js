/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import HomePage from './HomePage';

describe('Renders correctly', () => {
  it('should render correctly', () => {
    const output = shallow(
      <HomePage />,
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
