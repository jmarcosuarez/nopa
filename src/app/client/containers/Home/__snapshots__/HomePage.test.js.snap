// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly should render correctly 1`] = `
<Layout
  title="Welcome to Nopa!"
>
  <div
    className="main"
  >
    <div
      className="logo"
    >
      <Link
        className=""
        onlyActiveOnIndex={false}
        style={Object {}}
        title="Home"
        to="/"
      >
        <img
          alt="Logo"
          className="Nopa"
          src="test-file-stub"
        />
      </Link>
    </div>
    <h1>
      Your finances, in one place
    </h1>
    <p>
      Track all of your payments by connecting as many bank accounts as
      <br />
      you'd like to your Nopa account and get updates on your balance instantly.
    </p>
    <Button
      to="/choose-bank"
    >
      Get started
    </Button>
  </div>
  <div
    className="secondary"
  >
    <div>
      <div>
        <h1>
          There is no such things as "one size fits all" finance.
        </h1>
        <p>
          We were founded to make money simple and fair for everyone: whether you are looking for a loan, or to get better rewards for your investments.
        </p>
      </div>
      <div>
        <img
          alt="Shapes"
          className="shapes"
          src="test-file-stub"
        />
      </div>
    </div>
  </div>
</Layout>
`;
